assets/scss/views/_about.scss

```diff
@@ -36,14 +36,31 @@ main {
               @include flex-row;
 
               .member {
-                @include flex-row;
-
-                flex-basis: 50%;
-                flex-wrap: nowrap;
-                padding: 0 ($whitespace * 1.5) ($whitespace) 0;
+                border-radius: $radius;
+                display: flex;
+                flex-direction: row;
+                flex:  calc(50% - #{$whitespace});
+                margin: 0 $whitespace $whitespace 0;
```

Comment:
This margin seems very excessive -- there was already a consistent padding on these members (see line below), this additional margin results in very excessive whitespace that isn't needed?

| Live | Branch |
|-|-|
| ![image](https://user-images.githubusercontent.com/12371363/200179585-8bfa0ccb-e606-41df-92cf-09dacd2e2cbb.png) | ![image](https://user-images.githubusercontent.com/12371363/200179594-25c2c36b-09e3-4a75-83ae-c09d4e561369.png) |

Reply:
Good point!